src/Fantomas.Tests/KeepIfThenInSameLineTests.fs

```diff
@@ -0,0 +1,133 @@
+module Fantomas.Tests.KeepIfThenInSameLineTests
+
```

Comment:
@Bobface could you also add a test for when the MaxIfThenElseShortWidth is not exceeded.

Something like:
```fsharp
// Multiple conditionals with elif and else are indented at the same scope as the if:
if cond1 then e1
elif cond2 then e2
elif cond3 then e3
else e4
```
but every expression is shorter than MaxIfThenElseShortWidth.